src/function/table/arrow.cpp

```diff
@@ -193,6 +195,9 @@ static void arrow_scan_function(ClientContext &context, const FunctionData *bind
 		}
 
 		switch (output.data[col_idx].type.id()) {
+		case LogicalTypeId::SQLNULL:
+			FlatVector::Nullmask(output.data[col_idx]).all();
```

Comment:
Perhaps better to return a constant NULL here?

Reply:
aye